src/DependencyInjection/Extension.php

```diff
@@ -113,7 +113,7 @@ private function prepareDoctrineBundle(array $config, LoaderInterface $loader, C
 
         foreach ([
             PendingUserRepository::class => $classMapping[PendingUser::class] ?? null,
-            UserRepository::class => $classMapping[User::class],
+            UserRepository::class => $classMapping[User::class] ?? UserRepositoryInterface::class,
```

Comment:
can you do the same as for eav-bundle; https://github.com/msgphp/eav-bundle/blob/master/src/DependencyInjection/Extension.php#L61

Reply:
That is how it started. But (unless I missed something else I "fixed") it causes a further exception. Let me kick around a bit and see.

Reply:
i guess it's bit harder to for the user bundle yes. But this is mapping of "doctrine repo to entity class"`. So should be `?? User::class` otherwise.

originally it's a required entity (to be configured by the user), not doing so means it's optional by default basically and we need to act upon.

Reply:
Yup … and where I felt "uneasy" about my approach. IMHO what the bundle should *not* do is cause a fatal exception on installation, only on misconfiguration … and that one I couldn't figure out easily.